### PR TITLE
Add references of registry and image from imageutils

### LINKED ISSUE #76617: Fix: Use image util package to grab registry instead of static image path
Use image util package [[link](https://github.com/kubernetes/kubernetes/blob/master/test/utils/image/manifest.go)] to grab registry instead of using static image path to pull container images.

Reference: https://github.com/kubernetes/kubernetes/pull/75631#pullrequestreview-226247855

### PR DESCRIPTION
**Pull request by mgdevstack**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

 /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Updates codebase to use image registry from imageutils pkg.

**Which issue(s) this PR fixes**:

Fixes 
Should close the issue: #76617

**Special notes for your reviewer**:
 
**Does this PR introduce a user-facing change?**:

```release-note
NONE
```
/sig testing
cc @brahmaroutu

### CODE REVIEW

**Comment by imkin** [comment]
Please do not use "fixes" as it auto-closes issues. https://github.com/kubernetes/test-infra/issues/12240

**Comment by mgdevstack** [comment]
Thanks for linking `fixes:` issue @imkin . This PR updates all the image path to be picked from `imageutils` and were meant to close #76617. When the respective issue should be closed here remain in question and probably it would be an overhead for reviewer to have scan on whole codebase for such image path either before merging this PR or till the time Issue is open.

_Note: I have updated PR description, not to fix respective issue._

File: test/e2e/common/runtime.go [hunk 1]
```diff
@@ -353,28 +354,28 @@ while true; do sleep 1; done
...
 
 			It("should not be able to pull image from invalid registry [NodeConformance]", func() {
-				image := "invalid.com/invalid/alpine:3.1"
+				image := imageutils.GetE2EImage(imageutils.InvalidRegistryImage)
 				imagePullTest(image, false, v1.PodPending, true, false)
 			})
 
 			It("should not be able to pull non-existing image from gcr.io [NodeConformance]", func() {
-				image := "k8s.gcr.io/invalid-image:invalid-tag"
+				image := imageutils.GetE2EImage(imageutils.Invalid)
```
[the remark this answers is not in the document]
Comment (mgdevstack) [reply test/e2e/common/runtime.go]: Yeah, `const` might be another good option or rather `k8s.gcr.io/invalid-image:invalid-tag"` as it was. I was preferring to use `imageutils` for all valid and invalid aspects. As reviewer, if you insist to use const, we can update.

**Comment by mgdevstack** [comment]
Rebasing done.

**Pull request merged**